src/Spice86/Behaviors/InstructionPointerBehavior.cs

```diff
@@ -0,0 +1,80 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Spice86.ViewModels;
+
+namespace Spice86.Behaviors;
+
+/// <summary>
+/// Attached behavior for handling instruction pointer events in the disassembly view.
+/// This behavior encapsulates the UI-specific pointer event logic to improve separation of concerns.
+/// </summary>
+public class InstructionPointerBehavior
+{
```

Comment:
Formatting is not consistent with the editorconfig

Reply:
Fixed